_posts/2020-02-26-oblivious-parallel-ram.md

```diff
@@ -1,8 +1,9 @@
 ---
 layout: post
-title: Oblivious Parallel RAM
-subtitle: By George Wang and Haozhan Sun
+title: ABY
+subtitle: By Norah Tan and Qiulin Li
```

Comment:
Please do not change the OPRAM file.

Reply:
I have created a new commit below to change it to original